src/main/java/org/libertybikes/ai/service/AI.java

```diff
@@ -0,0 +1,25 @@
+package org.libertybikes.ai.service;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.libertybikes.ai.restclient.GameServiceClient;
+
+public class AI{
```

Comment:
I think we should use a different name for this class, perhaps `AIRegistration`?